### ISSUE TITLE
Intermittent test failure int ProxyForwardAuthDataTest.testForwardAuthData

### ISSUE THREAD

**Issue by merlimat** [apache/incubator-pulsar#1211]
https://builds.apache.org/job/pulsar-pull-request/org.apache.pulsar$pulsar-proxy/1581/testReport/junit/org.apache.pulsar.proxy.server/ProxyForwardAuthDataTest/testForwardAuthData/

```
Error Message

java.nio.channels.ClosedChannelException

Stacktrace

org.apache.pulsar.client.api.PulsarClientException: java.nio.channels.ClosedChannelException
	at org.apache.pulsar.client.impl.PulsarClientImpl.subscribe(PulsarClientImpl.java:215)
	at org.apache.pulsar.client.impl.PulsarClientImpl.subscribe(PulsarClientImpl.java:202)
	at org.apache.pulsar.proxy.server.ProxyForwardAuthDataTest.testForwardAuthData(ProxyForwardAuthDataTest.java:140)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
```

**Comment by merlimat** [comment]
@jai1  Can you take a look at this?

**Comment by merlimat** [comment]
Fixed in #1230
